app/models/events/poll_expired.rb

```diff
@@ -6,6 +6,7 @@ class Events::PollExpired < Event
   def self.publish!(poll)
     create(kind: "poll_expired",
            eventable: poll,
+           parent: poll.created_event,
```

Comment:
Seems like there's a sensible default of 
```
eventable.created_event
```

for this. In any case I'm simplifying the event creation process a bit in my branch, and will move this stuff over.